llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -164,6 +164,10 @@ def True : Predicate<"true">;
 class hasPTX<int version>: Predicate<"Subtarget->getPTXVersion() >= " # version>;
 class hasSM<int version>: Predicate<"Subtarget->getSmVersion() >= " # version>;
 
+// Explicit records for arch-accelerated SM versions
+def hasSM90a : Predicate<"Subtarget->getSmVersion() == 90"
```

Comment:
`getSmVersion()` is redundant here as `getFullSmVersion()` provides a more specific version comparison.

On a side note, now that we have to deal with `sm_??a` variants we may provide a generic `hasSMA<N>`.
`hasSM90a` here is fine for now.

Reply:
Fixed to use only getFullSmVersion() in the latest revision.